warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__rt_https.sql

```diff
@@ -0,0 +1,30 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__rt_feed_guideline_index') }}
+),
+
+rt_daily_url_index AS (
+    SELECT * FROM {{ ref('int_gtfs_rt__daily_url_index') }}
+),
+
+int_gtfs_quality__rt_https AS (
+    SELECT
+        idx.date,
+        idx.base64_url,
+        idx.feed_type,
+        CASE WHEN idx.feed_type = 'service_alerts' THEN {{ rt_https_service_alerts() }}
+             WHEN idx.feed_type = 'trip_updates' THEN {{ rt_https_trip_updates() }}
+             WHEN idx.feed_type = 'vehicle_positions' THEN {{ rt_https_vehicle_positions() }}
+        END AS check,
+        {{ best_practices_alignment() }} AS feature,
+        CASE
+            WHEN string_url LIKE 'https%' THEN "PASS"
+            WHEN string_url IS NOT null AND string_url NOT LIKE 'https%' THEN "FAIL"
+        END AS status,
+    FROM feed_guideline_index AS idx
+    JOIN rt_daily_url_index AS url_index
```

Comment:
Do we want a `LEFT JOIN` here? Because otherwise this is going to drop days where the URL isn't present.... Hmm. (Especially if we aren't using an equal row counts test, wondering if we are incurring some risk about grain here.)

Reply:
Ahh ok sorry for not following, I forgot about this. I was more concerned about cases like *since* November where there's a mismatch (like, say that URL A is in the guideline index on December 1, but not in the URL index.... Which now that I think about it I can't remember how possible that is).

Either way, I think that handling nulls with an NA is maybe preferable just because it's a bit more explicit? It's harder to troubleshoot a row that doesn't appear at all vs. a row that appears and explicitly has null data, if that makes sense